### PR TITLE
Updating location of logging playbook for master branch

### PR DESCRIPTION
**Pull request by ewolinetz**
Corresponding change for https://github.com/openshift/openshift-ansible/pull/6154
This should only be for master branch.

@stevekuznetsov 

CC @mtnbikenc @richm

### CODE REVIEW

**Comment by richm** [comment]
I'm not sure how aos-cd-jobs branches work, but afaik, the same sjb config runs against _all_ branches.  So if you want to use a different playbook location, you'll need to add some logic to the shell part to see if the new file exists, use it if it exists, or fall back to the old file if the new file doesn't exist.

**Comment by stevekuznetsov** [comment]
Yes, that is unfortunately the (sad) state of affairs @richm

**Comment by ewolinetz** [comment]
@mtnbikenc is this PR no longer needed then for https://github.com/openshift/openshift-ansible/pull/6154 ?

**Comment by mtnbikenc** [comment]
@ewolinetz This PR or a similar one is still needed for updating the location of the playbooks.  I updated https://github.com/openshift/openshift-ansible/pull/6154 so that it wouldn't break logging CI, and we could get this updated while having both new and old files in place.

**Comment by ewolinetz** [comment]
Updated to use the new logging playbook if its available or fall back to the prior location
PTAL

File: sjb/config/test_cases/test_branch_openshift_ansible_logging.yml [hunk 1]
```diff
@@ -144,6 +144,8 @@ extensions:
       title: "install origin-aggregated-logging"
       repository: "aos-cd-jobs"
       script: |-
+        logging_playbook='/usr/share/ansible/openshift-ansible/playbooks/openshift-logging/config.yml'
+        [[ ! -f ${logging_playbook} ]] && logging_playbook='/usr/share/ansible/openshift-ansible/playbooks/byo/openshift-cluster/openshift-logging.yml'
```
Comment (stevekuznetsov) [inline sjb/config/test_cases/test_branch_openshift_ansible_logging.yml]:
```sh
playbook_base='/usr/share/ansible/openshift-ansible/playbooks/'
if [[ -s "${playbook_base}openshift-logging/config.yml" ]]; then
    playbook="${playbook_base}openshift-logging/config.yml"
else
    playbook="${playbook_base}byo/openshift-cluster/openshift-logging.yml"
fi
```

**Comment by ewolinetz** [comment]
@stevekuznetsov addressed your nit

**Comment by stevekuznetsov** [comment]
/lgtm

@ewolinetz or @richm could you do the push?

**Pull request merged**